```
/home/runner/work/mod-phased-duels/mod-phased-duels/modules/mod-phased-duels/src/mod_phased_duels.cpp:70:36: fatal error: no member named 'SendNotification' in 'WorldSession'
   70 |         firstplayer->GetSession()->SendNotification("There are no free phases");
      |         ~~~~~~~~~~~~~~~~~~~~~~~~~  ^
1 error generated.
```

--- a/src/mod_phased_duels.cpp
+++ b/src/mod_phased_duels.cpp
@@ -67,8 +67,8 @@ void PhasedDueling::OnPlayerDuelStart(Player* firstplayer, Player* secondplayer)
         }
 
         // Couldnt find free unique phase
-        firstplayer->GetSession()->SendNotification("There are no free phases");
-        secondplayer->GetSession()->SendNotification("There are no free phases");
+        ChatHandler(firstplayer->GetSession()).SendNotification("There are no free phases");
+        ChatHandler(secondplayer->GetSession()).SendNotification("There are no free phases");
     }
 }
 

